Specify that the Content-type is text/plain not application/json

Alevale opened the pull request:
Specify that the content-type of the response is not JSON but plain text and therefore some libraries might not parse them properly when trying to parse the content of the requests that are made to the API.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Alevale committed:
Content-type is text/plain not application/json

Specify that the content-type of the response is not JSON but plain text and therefore some libraries might not parse them properly.
